crates/chia-tools/src/bin/benchmark-condition-costs.rs

```diff
@@ -0,0 +1,534 @@
+                    TEST_CONSTANTS.agg_sig_parent_additional_data.as_slice(),
+                ]
+                .concat(),
+            ),
+            cost: 1_200_000,
+        },
+        ConditionTest {
+            opcode: opcodes::AGG_SIG_PUZZLE,
+            args: &[pk_ptr, msg_ptr],
+            aggregate_signature: sign(
+                &sk,
+                [
+                    MSG1,
+                    puzzle_hash.as_slice(),
+                    TEST_CONSTANTS.agg_sig_puzzle_additional_data.as_slice(),
+                ]
+                .concat(),
+            ),
+            cost: 1_200_000,
+        },
+        ConditionTest {
+            opcode: opcodes::AGG_SIG_AMOUNT,
+            args: &[pk_ptr, msg_ptr],
+            aggregate_signature: sign(
+                &sk,
+                [
+                    MSG1,
+                    u64_to_bytes(100_u64).as_slice(),
+                    TEST_CONSTANTS.agg_sig_amount_additional_data.as_slice(),
+                ]
+                .concat(),
+            ),
+            cost: 1_200_000,
+        },
+        ConditionTest {
+            opcode: opcodes::AGG_SIG_PARENT_AMOUNT,
+            args: &[pk_ptr, msg_ptr],
+            aggregate_signature: sign(
+                &sk,
+                [
+                    MSG1,
+                    H1.as_slice(),
+                    u64_to_bytes(100_u64).as_slice(),
+                    TEST_CONSTANTS
+                        .agg_sig_parent_amount_additional_data
+                        .as_slice(),
+                ]
+                .concat(),
+            ),
+            cost: 1_200_000,
+        },
+        ConditionTest {
+            opcode: opcodes::AGG_SIG_PARENT_PUZZLE,
+            args: &[pk_ptr, msg_ptr],
+            aggregate_signature: sign(
+                &sk,
+                [
+                    MSG1,
+                    H1.as_slice(),
+                    puzzle_hash.as_slice(),
+                    TEST_CONSTANTS
+                        .agg_sig_parent_puzzle_additional_data
+                        .as_slice(),
+                ]
+                .concat(),
+            ),
+            cost: 1_200_000,
+        },
+        ConditionTest {
+            opcode: opcodes::AGG_SIG_PUZZLE_AMOUNT,
+            args: &[pk_ptr, msg_ptr],
+            aggregate_signature: sign(
+                &sk,
+                [
+                    MSG1,
+                    puzzle_hash.as_slice(),
+                    u64_to_bytes(100_u64).as_slice(),
+                    TEST_CONSTANTS
+                        .agg_sig_puzzle_amount_additional_data
+                        .as_slice(),
+                ]
+                .concat(),
+            ),
+            cost: 1_200_000,
+        },
+        ConditionTest {
+            opcode: opcodes::REMARK,
+            args: &[h1_pointer],
+            aggregate_signature: Signature::default(),
+            cost: 0,
+        },
+        //        ConditionTest {
```

Comment:
if you're not including this condition, I think you should remove the commented-out code